pgoapi/pgoapi.py

```diff
@@ -859,15 +859,23 @@ def incubate_eggs(self):
         if not self.EGG_INCUBATION_ENABLED:
             return
         if self.player_stats.km_walked > 0:
+            incubator_start_km_walked = self.player_stats.km_walked
+
             for incubator in self.inventory.incubators_busy:
-                incubator_egg_distance = incubator['target_km_walked'] - incubator['start_km_walked']
-                incubator_distance_done = self.player_stats.km_walked - incubator['start_km_walked']
+                if 'start_km_walked' in incubator:
+                    incubator_start_km_walked = incubator['start_km_walked']
```

Comment:
why not just `incubator_start_km_walked = incubator.get('start_km_walked', self.player_stats.km_walked)`